test/_RecorderTests.cpp

```diff
@@ -7,8 +7,12 @@
 #include <gtest/gtest.h>
 #include <rccl/rccl.h>
 
+#ifndef RCCL_EXPOSE_STATIC
 #include "RcclMockFuncs.hpp"
-//#include "TestBed.hpp"
+#else
+#include "info.h"
+#include "comm.h"
+#endif
```

Comment:
This code requires info.h and comm.h, so should always include them. It shouldn't need to know about RcclMockFuncs.hpp internals to make that decision.

```suggestion
#endif

#include "info.h"
#include "comm.h"
```

Reply:
Made modifications as per you suggestion.